app/api/margatsni/v1/base_v1.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'grape-swagger'
+Margatsni::V1::Validators::Length
```

Comment:
It is necessary for grape custom validators. As grape has problems with custom validators i used this line to mount it, otherwise it will rise UnknownValidator exception

You can view details here: https://github.com/ruby-grape/grape/issues/1178

Reply:
Ah, I see your point. Could you please provide some description of this in the comment. I guess just link to `ruby-grape/grape#1178` will be enough. Otherwise, it will definitely confuse each developer who would read this. Thanks